swagger/v3/api_spec.json

```diff
@@ -161,12 +161,12 @@
         }
       }
     },
-    "/api/v3/partnerships/ecf": {
+    "/api/v3/participants/ecf/transfers": {
```

Comment:
ah my bad we can't run swaggerise on v3 as it messes the order, we have to add manually sorry 😓 v1 and v2 we can run but not on v3